packages/react-components/src/components/AttachmentCard.tsx

```diff
@@ -77,68 +119,27 @@ export const _AttachmentCard = (props: _AttachmentCardProps): JSX.Element => {
     }
   }, [progress, showProgressIndicator, attachmentName, uploadStartedString, uploadCompletedString]);
 
-  const progressBarThicknessPx = 4;
-
   const containerClassName = mergeStyles({
     width: '12rem',
-    background: theme.palette.neutralLighter,
-    borderRadius: theme.effects.roundedCorner4,
-    border: `${_pxToRem(1)} solid ${theme.palette.neutralQuaternary}`,
-    cursor: 'pointer'
+    minWidth: '75%'
   });
 
-  const attachmentInfoWrapperClassName = mergeStyles({
-    padding: _pxToRem(12),
-    // To make space for the progress indicator.
-    paddingBottom: showProgressIndicator ? _pxToRem(12 - progressBarThicknessPx * 2) : _pxToRem(12)
-  });
-
-  const attachmentNameContainerClassName = mergeStyles({
-    paddingLeft: _pxToRem(4),
-    minWidth: '75%',
-    maxWidth: '75%'
-  });
-
-  const attachmentNameTextClassName = mergeStyles({
+  const fileNameContainerClassName = mergeStyles({
+    marginTop: _pxToRem(5),
+    width: '5.75rem',
     overflow: 'hidden',
     textOverflow: 'ellipsis',
-    lineHeight: 'normal',
-    whiteSpace: 'nowrap',
-    paddingRight: _pxToRem(4)
-  });
-
-  const actionIconClassName = mergeStyles({
-    cursor: 'pointer'
-  });
-
-  const progressIndicatorStyles: IStyleFunctionOrObject<IProgressIndicatorStyleProps, IProgressIndicatorStyles> = {
-    itemProgress: {
-      padding: `${_pxToRem(progressBarThicknessPx - 1)} 0`, // item progress height won't apply without an explicit padding
-      // To make the progress indicator border curve along the bottom of attachment card.
-      borderRadius: `0 0 ${theme.effects.roundedCorner4} ${theme.effects.roundedCorner4}`
-    },
-    progressBar: {
-      height: _pxToRem(progressBarThicknessPx)
+    span: {
+      whiteSpace: 'nowrap'
     }
-  };
+  });
 
   return (
     <div data-is-focusable={true}>
       <Announcer announcementString={announcerString} ariaLive={'polite'} />
-      <Stack
-        className={containerClassName}
-        onClick={() => {
-          props.actionHandler?.();
-        }}
-      >
-        <Stack
-          horizontal
-          horizontalAlign="space-between"
-          verticalAlign="center"
-          className={attachmentInfoWrapperClassName}
-        >
-          <Stack>
-            {/* We are not using <ChatCompositeIcon /> here as we currently do not support customizing these attachmenttype icons. */}
+      <Card className={containerClassName} size="small" role="listitem">
```

Comment:
`Card` is FluentUI v9 component but `containerClassName` is FluentUI v8 styles. We should **always** use v8 component with v8 styles and v9 components with v9 styles and **never** mix them up. 
This is needed because FluentUI components already have some internal styles and styles from different versions can't be merged (as different css libraries are used) and conflict of styles appears (when browser doesn't know which styles takes priority).